packages/dds/tree/src/core/schema-stored/format.ts

```diff
@@ -52,7 +52,7 @@ export const TreeNodeSchemaIdentifierSchema = brandedStringType<TreeNodeSchemaId
 
 const FieldSchemaFormatBase = Type.Object({
 	kind: FieldKindIdentifierSchema,
-	types: Type.Optional(Type.Array(TreeNodeSchemaIdentifierSchema)),
+	types: Type.Array(TreeNodeSchemaIdentifierSchema),
```

Comment:
Changing the persisted format is high risk, and requires very careful consideration.

It should be impossible for documents using undefined here (meaning any) to exist and if they did, such documents are impossible to view due to being incompatible with the all possible view schema. That is my justification anyway, but reviewers should double, and triple check this reasoning before approving a persisted format change!